src/js/components/Card.js

```diff
@@ -74,9 +75,36 @@ export default class Card extends Component {
     return videoLayer;
   }
 
+  _renderParagraph (contents, textSize, type) {
+    if (typeof contents === 'string') {
+      return (
+        <Paragraph
+          className={`${CLASS_ROOT}__${type}`}
+          size={textSize}
+          margin="none"
+        >
+          {contents}
+        </Paragraph>
+      );
+    } else if (Array.isArray(contents)) {
```

Comment:
I see this supports contents being an Array but the prop types for both are just string. Do we need the Array support?